tools/profile_endpoints.py

```diff
@@ -19,17 +19,17 @@
  - getConceptHierarchy: Profiles the performance of the getConceptHierarchy endpoint by sending
  GET requests with different concept IDs.
 
- - searchConcepts: Profiles the performance of the searchConcepts endpoint by sending GET requests
+ - enumerateConcepts: Profiles the performance of the enumerateConcepts endpoint by sending GET requests
 with different domain IDs and search texts.
 
- - getConcepts: Profiles the performance of the getConcepts endpoint by sending GET requests with
+ - getConceptChildren: Profiles the performance of the getConceptChildren endpoint by sending GET requests with
  different concept IDs.
 """
 
 # Constants
 DATASETS = {
```

Comment:
Should this be SNAPSHOTS now?

Reply:
Oh, good catch! Thanks - yes, I should change the argument to references snapshots as well.